### PR TITLE
export toHaveHtml to type definition.

### PR DESCRIPTION
**Pull request by haishengwu-okta**
An follow up of https://github.com/unindented/custom-jquery-matchers/pull/38

Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 coverage/
 dist/
 node_modules/
+.log/
```
Comment (unindented) [inline .gitignore:4]: What is producing this `.log` folder?

Comment (haishengwu-okta) [reply .gitignore:4]: Emacs LSP plugin for JavaScript/TypeScript

Comment (unindented) [reply .gitignore:4]: I prefer to keep all editor-specific stuff out of project-specific `.gitignore` files. That stuff should go in people's global `.gitignore`.

Would you mind removing?

Comment (haishengwu-okta) [reply .gitignore]: Removed. Thanks the feedback!

**Comment by haishengwu-okta** [comment]
hi @unindented any more feedback or you plan to merge at some point? thanks!

**Comment by unindented** [comment]
@haishengwu-okta sorry about that, got distracted by life.

I'll publish a new version now.

**Comment by unindented** [comment]
Published v2.1.1. Thanks for your help!

**Pull request merged**